github/ci/prow-deploy/molecule/default/prepare.yml

```diff
@@ -104,7 +104,7 @@
 
     - name: copy production kustomize files
       synchronize:
-        src: '{{ project_infra_root }}/github/ci/prow-deploy/kustom/overlays/ibmcloud-production/{{ item }}'
+        src: '{{ project_infra_root }}/github/ci/prow-deploy/kustom/overlays/kubevirt-prow-control-plane/{{ item }}'
```

Comment:
nit: this and the other hard coded cluster name change could very well be a variable, however we should leave that for another occasion.